docs/getting_started/download.md

```diff
@@ -10,6 +10,10 @@ More combinations will be made available as our testing matrix continues to matu
 
 The latest packages can be downloaded from our APT and YUM repositories hosted at [https://packages.irods.org](https://packages.irods.org).
 
+## Packages and Platform End-of-Life (EOL) Policy
```

Comment:
let's bump it one level in (###) so that the page still only has two sections, as mentioned in the sentence at the top of the page.

We could also put this language in the 'Binary Distributions' of the irods.org/download page...

Reply:
Will do.

Reply:
Added extra pound to header.

Will add to the irods.org repo once this PR is merged.